pages/batch-bitly.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <main class="space-y-4">
+    <h1 class="text-xl font-bold font-mono ">
+      Batch Create Bitly Links
+    </h1>
+    <UFormGroup
+      label="Bitly Access Token"
+      :required="true"
+    >
+      <UInput
+        v-model="bitlyToken"
+        class="font-mono"
+        :required="true"
```

Comment:
Can we set `autocomplete="off"` here to prevent browser storing secret? or set type=password